### PR TITLE
Bug 1136900 - settings panel

### PR DESCRIPTION
**Pull request by ncalexan**

### CODE REVIEW

**Comment by darrinhenein** [comment]
Functionally it looks great, can we move the Done button to the left side? See http://invis.io/M52BE4WD3.

File: Client/Frontend/SettingsTableViewController.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```
Comment (thebnich) [inline Client/Frontend/SettingsTableViewController.swift]: Nit: Indentation (if you're interested, I made a [template](https://www.dropbox.com/s/9l38op33lzhx408/Templates.zip?dl=0) that you can extract to `~/Library/Developer/Xcode` which includes the Moz header in all new Swift files).

**Comment by thebnich** [comment]
Looks good to me. Glad we're finally getting rid of this code!

File: Client/Frontend/SettingsTableViewController.swift (hunk 1 above)
Comment (ncalexan) [inline Client/Frontend/SettingsTableViewController.swift]: On Thu, Feb 26, 2015 at 9:36 AM, Brian Nicholson <notifications@github.com>
wrote:

> In Client/Frontend/SettingsTableViewController.swift
> <https://github.com/mozilla/firefox-ios/pull/183#discussion_r25446652>:
>
> > @@ -0,0 +1,17 @@
> > +/* This Source Code Form is subject to the terms of the Mozilla Public
> > +* License, v. 2.0. If a copy of the MPL was not distributed with this
> > +* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
>
> Nit: Indentation (if you're interested, I made a template
> <https://www.dropbox.com/s/9l38op33lzhx408/Templates.zip?dl=0> that you
> can extract to ~/Library/Developer/Xcode which includes the Moz header in
> all new Swift files).
>
Ta, I caught this after pushing as well.  I copy-paste to start all my new
files and Xcode really screws me.

**Pull request merged**